public/script.js

```diff
@@ -1458,21 +1458,8 @@ async function Generate(type, automatic_trigger, force_name2) {
             storyString += appendToStoryString(charPersonality, power_user.disable_personality_formatting ? '' : 'Personality: ');
             storyString += appendToStoryString(Scenario, power_user.disable_scenario_formatting ? '' : 'Scenario: ');
         } else {
-            if (charDescription !== undefined) {
```

Comment:
I cannot grasp why **personality** manipulation depends on **description** being defined. I think it is simply a copy-paste error introduced in 7b87227cd2f73241b54a52f50fdc8ef4c73fc81b, so this condition is wrong and `appendToStoryString()` can be used here just fine (like for Pygmalion a few lines above).